docs/src/dev-guide/components-core/regex-utils.md

```diff
@@ -62,6 +62,19 @@ For a detailed description on the options order and usage, see the
   * Turn `.*` into `*`
   * Turn `.+` into `?*`
   * E.g. `abc.*def.ghi.+` will get translated to `abc*def?ghi?*`
+* Metacharacter escape sequences
+  * Append the escaped metacharacters literally to the wildcard output string.
```

Comment:
Lets re-word this a bit to something like:

An escaped regex metacharacter is treated as a literal and appended to the wildcard output string.

Similar to other places, this is written as the "append" is happening "literally" when we mean the metacharacter is treated as a literal character.